deps/rabbit/src/rabbit_disk_monitor.erl

```diff
@@ -274,12 +250,26 @@ get_disk_free(Dir, {win32, _}, #state{win32 = W32State}) ->
             {ok, Free} = win32_get_disk_free_dir(Dir),
             Free;
         DriveLetter ->
-            case catch win32_get_disk_free_pwsh(DriveLetter, W32State) of
-                {ok, Free1} -> Free1;
-                PwshNotOk ->
-                    rabbit_log:error("could not determine disk free: ~p", [PwshNotOk]),
-                    exit(could_not_determine_disk_free)
-            end
+            % Note: yes, "$ " is the $char sequence for an ASCII space
+            F = fun([D, $:, $\\, $  | _]) when D =:= DriveLetter ->
```

Comment:
A more readable version of `$ ` is `$\s`. Perhaps because I always saw the latter and never the former.

Reply:
I have rebased and force pushed it to eliminate some test failures addressed in master.

Reply:
Let me take care of the escaping sequence.

Reply:
FTR,

``` erl
"$ " =:= "$\s".
%% => true
```